### PR TITLE
Add filteringCharset = 'UTF-8' to gradle

### PR DESCRIPTION
**Pull request by Siedlerchr**
When I read it correct in the doc, this should help fixing the Unicode problems in #1178 
I tested locally and it seemed working for me. 

>It's a good practice to specify the charset when reading and writing the file, using the filteringCharset property. If not specified, the JVM default charset is used, which might not match with the actual charset of the files to filter, and might be different from one machine to another. 

https://docs.gradle.org/current/userguide/working_with_files.html#sec:filtering_files

### CODE REVIEW

**Comment by grimes2** [comment]
On my system the encoding is now working.

encoding

I changed
filteringCharset = 'UTF-8'
to
filteringCharset = 'ISO-8859-1'

JabRef 3.7dev
windows 10 10.0 amd64 
Java 1.8.0_101

**Comment by koppor** [comment]
Checking @Siedlerchr's changes, it works. @grimes2 I don't get, why ISO-8859-1 works at your side and UTF-8 doesn't. I just merge this in and we can continue evaluating afterwards.

File: build.gradle [hunk 1]
```diff
@@ -47,7 +47,7 @@ install4j {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.14.1'
+    gradleVersion = '3.0'
```
Comment (koppor) [inline build.gradle]: This is not necessary anymore. See https://gradle.org/blog/gradle-3-0-m1-unleash-the-daemon/

`./gradlew wrapper --gradle-version 3.0` has the same effect as `./gradlew wrapper` with that config.

I'll just change it directly in the master branch.

**Comment by grimes2** [comment]
It seems that this bug is not fixed.
wrong_encoding

JabRef 3.7dev--snapshot--2016-09-11--master--f5ed5ac
windows 10 10.0 amd64 
Java 1.8.0_101

**Comment by Siedlerchr** [comment]
Yes, you are right. Windows 7 x64 the same.

**Comment by Siedlerchr** [comment]
I think I found the reason for it.  Property.load assumes automatically ISO... charset.
I try with wrapping it in an `InputStreamReader `
http://stackoverflow.com/a/17852323

**Comment by koppor** [comment]
Strange, that it was not fixed at your side, here on Win 10, 64bit, the umlauts were displayed correctly.

**Comment by Siedlerchr** [comment]
@koppor  Nope, if you look closely in the middle you have someone named `Krzysztof A. Kościuszkiewicz`
 where the s -Char is not displayed correctly, only a question mark => Latin 1 (supports only a subset of chars). (I recently read an article about a system in the Swiss Government which had the same problems...)

**Pull request merged**